src/components/track.tsx

```diff
@@ -3,49 +3,84 @@ import * as React from "react";
 import { accentColor } from "../constants";
 import { usePublicFileURL } from "../hooks/use-public-file-url";
 
+const TitleContainer = styled.div`
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4.5rem;
+  overflow: hidden;
+  transition: 0.5s ease;
+
+  background-color: rgba(0, 0, 0, 0.75);
+
+  -webkit-backdrop-filter: blur(10px);
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+
+  z-index: 0;
+  cursor: default;
+`;
+
 const ImageContainer = styled.div`
   position: relative;
 
   display: block;
-  width: 100%;
   height: 15rem;
   overflow: hidden;
   margin-bottom: 1rem;
 
   border-radius: 5px;
 
   background-size: 100% auto;
-  background-position: center center;
   background-color: white;
+  background-position: center center;
```

Comment:
why move this property around?

Reply:
That is a very good question, probably happened as I was moving properties around in order to split stuff apart. Will be resolved in the following commit.